File: src/main/java/com/pCarpet/controller/LegislationController.java
package com.pCarpet.controller;

import javax.servlet.http.HttpServletRequest;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;


@Controller
@RequestMapping("/Legislation")
public class LegislationController {
	
	@Autowired
	public LegislationController() {
		
	}
	
	@RequestMapping(value = "/legislations", method = RequestMethod.GET)
	public String getLegislations(HttpServletRequest request, Model model ) {
		return "legislation";
	}

}

File: PCarpet/Controllers/LegislationController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace PCarpet.Controllers
{
    public class LegislationController : Controller
    {
        // GET: Legislation
        public ActionResult Index()
        {
            return View();
        }

        public ActionResult Legislations()
        {
            return View();
        }
    }
}